app/models/user/reputation_tokens/code_contribution_token.rb

```diff
@@ -1,18 +1,18 @@
 class User::ReputationTokens::CodeContributionToken < User::ReputationToken
-  params :repo, :pr_id
+  params :repo, :pr_id, :pr_number
   category :building
   reason :contributed_code
   levels %i[minor regular major]
   values({ minor: 5, regular: 10, major: 15 })
 
   def guard_params
-    "PR##{repo}/#{pr_id}"
+    "PR##{pr_id}"
   end
 
   def i18n_params
     {
       repo: repo,
-      pr_id: pr_id
+      pr_number: pr_number
```

Comment:
This also makes more sense in the i18n text, as we are actually referring to the PR number and not its ID.